### DISCUSSION TITLE
绝区零

### DISCUSSION THREAD

**Discussion by kires-tee** [share121/inter-knot#2368] [其他]
我问下绝区零的深渊你们都能打过么，我看抖音都轻轻松松七个s，我怎么打死就打过四个，还都是A、B
<img src="https://img-static.mihoyo.com/communityweb/upload/70b5b0c9543ee5080c6ae92fd2797892.png" alt="心情复杂" width="75" >

**Comment by bananice12138** [comment]
我也没打过几个，我到现在的稳定防线的10个都没打过去呢，再说了能发抖音的基本都是打的好才会发出来，不要被强度焦虑裹挟了玩游戏的心情哦<img src="https://upload-bbs.miyoushe.com/upload/2023/03/09/3173bc3e69fcfebecfda741340c9ad22_6841250310756275678.png" alt="开拓者 吃瓜ing" width="75" >

**Comment by Genshin-Neuvillette** [comment]
你和手法大帝比当然是比不过的，但是不必担心。我打10把攻坚战才到A<img src="https://img-static.mihoyo.com/communityweb/upload/4a102d21856350263e83278ac8c48526.png" alt="吃惊" width="75" >

**Comment by Xy9223** [comment]
实不相瞒，限时的我才过两三关

**Comment by chihung3927** [comment]
首先你得主C60,其次盘子得刷到一般水准
